Draft preamble for 6.2 release notes (#12065)
gradle/gradle: subprojects/docs/src/docs/release/notes.md

The Gradle team is excited to announce Gradle @version@.

> This release adds [built-in dependency checksum and signature verification](#dependency-verification) to keep builds safe, provides a [shareable read-only dependency cache](#shared-dependency-cache) and contains [helpful documentation to avoid deprecations in your build](#deprecation-messages).

Reviewer: How about "shareable read-only dependency cache **to accelerate ephemeral builds**"?

Author: removed

> This release adds [built-in dependency checksum and signature verification](#dependency-verification) to keep builds safe, provides a [shareable read-only dependency cache](#shared-dependency-cache) and contains [helpful documentation to avoid deprecations in your build](#deprecation-messages).

Author: how about?
> emits [helpful documentation links when you have deprecations in your build](#deprecation-messages)

> This release adds [built-in dependency checksum and signature verification](#dependency-verification) to keep builds safe, provides a [shareable read-only dependency cache](#shared-dependency-cache) and contains [helpful documentation to avoid deprecations in your build](#deprecation-messages).

Reviewer 2: the part I'm struggling with is the subject to verb relationship. The release itself doesn't print or emit links. :/

> This release adds [built-in dependency checksum and signature verification](#dependency-verification) to keep builds safe, provides a [shareable read-only dependency cache](#shared-dependency-cache) and contains [helpful documentation to avoid deprecations in your build](#deprecation-messages).

Reviewer 2: it doesn't look bad though. let's mark as resolved.

Gradle 6.2 ships with [dependency verification](userguide/dependency_verification.html).
Dependency verification is a major step towards a safer ecosystem by making it possible to verify both the checksums and the signatures of dependencies and plugins used during a build.

Please refer to the [user manual](userguide/dependency_verification.html) to figure out how to enable dependency verification.

We would like to give special thanks to [Vladimir Sitnikov](https://github.com/vlsi) for his feedback and inspiration.
A lot of the work on this feature is, in particular, available to previous versions of Gradle via his [Checksum Dependency Plugin](https://plugins.gradle.org/plugin/com.github.vlsi.checksum-dependency).

<a name="shared-dependency-cache"></a>
## Shared dependency cache

Improving on [relocatable dependency caches introduced in the previous release](https://docs.gradle.org/6.1.1/release-notes.html#ephemeral-ci:-reuse-gradle's-dependency-cache), Gradle 6.2 now offers the ability to **share** a dependency cache between multiple instances.

> In the context of ephemeral builds on disposable containers, this makes it possible to have a single, shared, directory between containers which contains most, if not all, the dependencies required by all builds.

Reviewer 2: ```suggestion
In the context of ephemeral builds on disposable containers, this makes it possible to have a single, shared directory that contains the dependencies required by all builds.
```

Reviewer 2: Since the most if not all part is not explained in the release notes, I'm recommending that you remove it.

Author: done
